### ISSUE TITLE
Remove Python 2.6 relevant test from twisted/trial/test/test_pyunitcompat.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8348]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8348|
|Type|enhancement|
|Created|2016-05-22 19:08:52Z|
|Branch|https://github.com/twisted/twisted/tree/8348-rodrigc-py26|

we no longer support py 2.6 so these tests can be removed as they will never be executed

```
[SKIPPED]
This test is only relevant to Python 2.6
```

<details><summary>Searchable metadata</summary>

```
trac-id__8348 8348
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__8348_rodrigc_py26 8348-rodrigc-py26
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1463944132986387 1463944132986387
changetime__1466963089588967 1466963089588967
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/225

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Great cleanup. Thanks. Please merge!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset eeeacc946269ef9db463c8c791f37dc3e753eb5f
```
#!CommitTicketReference repository="" revision="eeeacc946269ef9db463c8c791f37dc3e753eb5f"
Merge 8348-rodrigc-py26: Remove test inside trial which is relevant to Python 2.6

Author: rodrigc
Reviewer: adiroiban, alex
Fixes: #8348
```
